docs/4.templates/2.template-custom-webchat.mdx

```diff
@@ -418,6 +418,6 @@ Once the specified inputs are matched:
 > We suggest you to check the [full template](https://github.com/hubtype/botonic/tree/master/packages/botonic-cli/templates/custom-webchat) which offers a bunch of examples to get started customizing the Webchat components.
 
 <div style={{display:"flex", justifyContent:"space-between"}}>
-  <PrevButton title="Dynamic Carousel" href="/tutorials-and-examples/dynamic-carousel" />
-  <NextButton title="Testing Your Components" href="/tutorials-and-examples/testing-your-components" />
+  <PrevButton title="List of Templates" href="/templates/templates" />
+  <NextButton title="Childs" href="/templates/childs" />
```

Comment:
childs should be children in English right? not sure if now is the best time to change this

Reply:
Indeed. It can be updated later on when reworking templates workflow.